lib/run/index.js

```diff
@@ -42,7 +42,8 @@ var _ = require('lodash'),
         json: require('../reporters/json'),
         junit: require('../reporters/junit'),
         progress: require('../reporters/progress'),
-        emojitrain: require('../reporters/emojitrain')
+        emojitrain: require('../reporters/emojitrain'),
+        postman: require('../reporters/postman')
```

Comment:
Are we sure the usage should be `-r postman`? What happened to `--publish`?

If yes, why it's named `postman`? 🤔

Reply:
Using the -r postman follows the reporter convention which seems much better. Deprecating --publish would have been difficult (if needed).
Also, later when we add other configurable params (like timeout, retries), this approach would prevent bloating of newman cli options. They being specific to the reporter wont really belong as params of newman which doesnt use these (better separation of concerns on the interface)

Why `postman`: We went with postman as the data is being pushed to postman servers. (similar to other external reporters mentioned here (https://postmanlabs.atlassian.net/wiki/spaces/RUN/pages/3890544729))

We also considered `cloudpublish/postmanpublish/postmancloud` but `postman` seemed better.
Please let me know if you have any suggestions on this.